src/Altinn.App.Api/Controllers/ActionsController.cs

```diff
@@ -107,9 +110,13 @@ public ActionsController(IAuthorizationService authorization, IInstanceClient in
             });
         }
 
-        var result = await actionHandler.HandleAction(userActionContext);
+        UserActionResult result = await actionHandler.HandleAction(userActionContext);
```

Comment:
This is the main change in this class, adding support for redirect results in UserActions.
Rest is just variable type nitpicks.